charts/backingservices/charts/srs/templates/srsservice_networkpolicy.yaml

```diff
@@ -0,0 +1,44 @@
+{{ if .Values.enabled }}
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
```

Comment:
Do we want to put this in the client managed cloud Helm charts? Ping me to discuss if that doesn't make sense.